### PR TITLE
feat: orgBrowser test in redhat

### PR DESCRIPTION
**Pull request by CristiCanizales**
- Adds orgBrowser test file
- Implement test for Retrieve and Open Source action
- Implement test for Retrieve This Source from Org
- Import after from vscode-extension-tester so screenshots work

@W-17041496@
@W-17042844@

[Test run](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/11503380465) ✅

### CODE REVIEW

**Review by peternhale — APPROVED** [review]

File: test/utilities/orgBrowser.ts [hunk 1]
```diff
@@ -4,25 +4,31 @@
...
-    label
-  );
-  // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-  expect(orgBrowserLabelEl).to.be.ok;
+export async function verifyOrgBrowserIsOpen(): Promise<void> {
+  const orgBrowser = new SideBarView();
+  const titlePart = orgBrowser.getTitlePart();
+  const title = await titlePart.getTitle();
+  expect(title).to.equal('ORG BROWSER: METADATA');
 }
```
Comment (daphne-sfdc) [inline test/utilities/orgBrowser.ts:22]: Great improvement! 😀

File: test/specs/orgBrowser.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    expect(apexClassesItem).to.not.be.undefined;
+    const refreshComponentsButton = (await apexClassesItem?.findElements(By.css('a.action-label')))![1];
+    expect(refreshComponentsButton).to.not.be.undefined;
+    await refreshComponentsButton?.click();
+    await utilities.pause(utilities.Duration.seconds(2));
+    const myClassLabelEl = await utilities.findTypeInOrgBrowser('MyClass');
+    expect(myClassLabelEl).to.not.be.undefined;
+  });
+
+  step('Retrieve This Source from Org', async () => {
```
Comment (daphne-sfdc) [inline test/specs/orgBrowser.e2e.ts:106]: You got this step to work! 🎉

File: test/specs/orgBrowser.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    await modalDialog.pushButton('Overwrite');
+
+    const successNotificationWasFound = await utilities.notificationIsPresentWithTimeout(
+      'SFDX: Retrieve This Source from Org successfully ran',
+      utilities.Duration.FIVE_MINUTES
+    );
+    expect(successNotificationWasFound).to.equal(true);
+  });
+
+  step('Retrieve and Open Source', async () => {
```
Comment (daphne-sfdc) [inline test/specs/orgBrowser.e2e.ts:128]: And this one too! 💃

**Review by daphne-sfdc — APPROVED** [review]
Thank you for the great improvements @CristiCanizales! 🤩

**Pull request merged**